torch/ao/quantization/fx/prepare.py

```diff
@@ -104,24 +104,17 @@ def node_arg_is_weight(node: Node, arg: Any) -> bool:
                 return True
     return False
 
-CONV_OPS_WITH_BIAS = {
-    torch.nn.functional.conv1d,
-    torch.nn.functional.conv2d,
-    torch.nn.functional.conv3d,
-}
-CONV_BIAS_ARG_INDEX = 2
-
 def node_arg_is_bias(node: Node, arg: Any) -> bool:
-    if isinstance(node, Node) and node.op == 'call_function':
-        if node.target in CONV_OPS_WITH_BIAS:
-            for i, node_arg in enumerate(node.args):
-                if arg is node_arg and i == CONV_BIAS_ARG_INDEX:
-                    return True
-        elif node.target in FUNCTIONAL_OPS_WITH_BIAS:
-            for kwarg_name, kwarg_value in node.kwargs.items():
-                if kwarg_name == 'bias' and arg is kwarg_value:
-                    return True
-    return False
+    if not isinstance(node, Node) or node.op != 'call_function':
+        return False
+
+    assert not isinstance(node.target, str), "expected a function target"
```

Comment:
this is a check function, should we have assert here?

Reply:
I though this was necessary for mypy, but it seems to be passing without it so I guess not.